nextcord/http.py

```diff
@@ -341,6 +344,14 @@ async def request(
                                 bucket,
                                 delta,
                             )
+                            self._dispatch(
+                                "http_ratelimit",
+                                5,
+                                0,
```

Comment:
Why are these hard coded?

Reply:
Well I made them hard coded because the limit is a static value and if we get ratelimited the remaining value should be 0. Maybe since these are static I shouldn't provide them in the event, especially the remaining value.

Reply:
I was not aware of these. Can you link the resources where the limit is stated to be 5 for all routes?

Reply:
Well I assumed that because Nextcord/discord.py treats it like a static value in the `request` function (https://github.com/nextcord/nextcord/blob/master/nextcord/http.py#L308).

Reply:
Thats not the ratelimit...? Thats how many times to retry a request if it fails for any reason

[Ratelimits](https://discord.com/developers/docs/topics/rate-limits) are dynamic, and different per bucket + discord says not to hard code them so we shouldn't where possible. Even on the examples there are multiple where the `limit` is different

Reply:
Resolved in 79e39a4.